### PR TITLE
feat: Apply copy update on /server/download/thank-you

### PR DESCRIPTION
**Pull request by britneywwc**
## Done

- Apply copy updates
- Make CLI cheatsheet download link gated

## QA

- Go to https://ubuntu-com-14584.demos.haus/download/server/thank-you?version=24.04.1&architecture=amd64&lts=true
- Check that it matches [copy doc](https://docs.google.com/document/d/1S5zY7HN_Q8eQWch0mCEfLICfj8pW1bct9_wGArCNIqc/edit?tab=t.0)
- Click on `Download the CLI cheat sheet` CTA button
  - Submit form, check that it redirects to CLI pdf

## Issue / Card

Fixes [WD-17869](https://warthogs.atlassian.net/browse/WD-17869)

## Screenshots

[If relevant, please include a screenshot.]

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

[WD-17869]: https://warthogs.atlassian.net/browse/WD-17869?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: templates/download/server/thank-you.html [hunk 1]
```diff
@@ -32,71 +32,24 @@ <h1>Thank you for downloading Ubuntu Server {{ version }}{{ " LTS" if lts }}</h1
...
-  </section>
-
-  {% include 'download/shared/_server_weekly_news.html' %}
-
-  <section class="p-section">
-    <hr class="p-rule is-fixed-width" />
-    <div class="row--50-50">
-      <div class="col">
+        {% include 'download/shared/_server_weekly_news.html' %}
         <h2>Ubuntu CLI cheatsheet</h2>
```
Comment (muhammad-ali-pk) [inline templates/download/server/thank-you.html:36]:
```suggestion
        <h2>Get Ubuntu Server pro tips</h2>
```

Comment (muhammad-ali-pk) [reply templates/download/server/thank-you.html:36]: I believe this heading should be changed, as per copydoc.

**Review by muhammad-ali-pk — CHANGES_REQUESTED** [review]

File: templates/download/server/thank-you.html (hunk 1 above)
Comment (thisisrichardferreira) [reply templates/download/server/thank-you.html:36]: Hi @muhammad-ali-pk, looks like the copydoc wasn't as up to date as the web page, we moved towards "CLI cheatsheet" — I made the adjustment and added a comment on the copydoc.

**Review by muhammad-ali-pk — APPROVED** [review]
LGTM!

**Pull request merged**